.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!--
+Hey! You're PRing? Cool! Please have a look at the below checklist. It's here to help both you and the maintainers remember some things. For more details, please see our contribution guide.
+-->
+
+### Checklist for PRs
+
+- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` if appropriate 
+- [ ] Created new or adapted existing unit tests
+- [ ] Added myself to `AUTHORS.rst` (optional)
+
+
+### If the PR contains API changes (delete if not relevant)
```

Comment:
```suggestion
### When the PR contains API changes (delete if not relevant)
```

Reply:
When = time, If = conditional ?!

Reply:
yeah but you already have the condition in the parenthesis. Also, you can use when to introduce something that is going to happen/is likely to happen. It is closely bound to being in the future but that is also true since the future happens after the PR person reads the line :)

Reply:
```suggestion
### If the PR contains API changes (otherwise, you can delete this passage)
```

alternative

Reply:
> yeah but you already have the condition in the parenthesis. Also, you can use when to introduce something that is going to happen/is likely to happen. It is closely bound to being in the future but that is also true since the future happens after the PR person reads the line :)

Reading this feels a bit like whatching Interstellar ;P I'll go with the second suggestion :D